processing/src/main/java/io/druid/segment/column/DoubleColumn.java

```diff
@@ -45,12 +52,18 @@ public int getLength()
   @Override
   public ColumnCapabilities getCapabilities()
   {
-    return CAPABILITIES;
+    return nullValueBitmap.isEmpty() ? CAPABILITIES : CAPABILITIES_WITH_NULL;
   }
 
   @Override
   public GenericColumn getGenericColumn()
   {
-    return new IndexedDoublesGenericColumn(column.get());
+    return new IndexedDoublesGenericColumn(column.get(), nullValueBitmap);
+  }
+
+  @Override
+  public ImmutableBitmap getNullValueBitmap()
+  {
+    return null;
```

Comment:
how is this working if it returns null ? not sure am getting this?

Reply:
It was an unused method,  was earlier added before columnSelectors were refactored in another PR, removed this method.